website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -163,9 +167,11 @@ To peer Kubernetes clusters running Consul, you need to create a peering token a
     $ kubectl --context $CLUSTER2_CONTEXT apply --filename dialer.yaml 
     ```
 
-## Export services between clusters
+### Export services between clusters
+
+~> **Tip**: The examples in this section demonstrate how to export a service named `backend`. You should change instances of `backend` in the example code to the name of the service you want to export.
```

Comment:
```suggestion
The examples described in this section demonstrate how to export a service named `backend`. You should change instances of `backend` in the example code to the name of the service you want to export.
```
I wouldn't characterize this information as a tip. It's salient information for understanding the code snippets